addons/mail/static/src/components/discuss_sidebar_category_item/discuss_sidebar_category_item.js

```diff
@@ -110,19 +135,18 @@ export class DiscussSidebarCategoryItem extends Component {
         ev.stopPropagation();
     }
 
-    _onClickLeaveGroupChat() {
-        this.thread.members.length === 1 ? this.thread.unsubscribe() : this.state.hasLeaveConfirmDialog = true;
-    }
-
     /**
      * @private
      * @param {MouseEvent} ev
      */
     async _onClickLeave(ev) {
         ev.stopPropagation();
-        if (this.categoryItem.channel.creator === this.env.messaging.currentUser) {
+        if (this.categoryItem.channel.channel_type !== 'group' && this.categoryItem.channel.creator === this.env.messaging.currentUser) {
             await this._askAdminConfirmation();
         }
+        if (this.categoryItem.channel.channel_type === 'group' && this.categoryItem.channel.members.length > 1) {
```

Comment:
This is part of the spec to not ask confirmation if there is only 1 member?
I wonder if we shouldn't completely delete the channel and all its message in that case, instead of just leaving it.